depthimage_to_laserscan:
  ros__parameters:
    scan_time: 0.033
    range_min: 0.45
    range_max: 10.0
    scan_height: 1
    output_frame: "camera_depth_frame"
